style/header.css

```diff
@@ -66,54 +62,39 @@
   align-items: center;
 }
 
-.main-content-menu ul {
+#search {
+  font-size: 1rem;
   text-transform: uppercase;
   margin: 0;
-  display: flex;
-  justify-content: space-evenly;
-  flex-wrap: wrap;
-}
-
-.main-content-menu li {
-  padding: 35px 20px 18px;
-  color: var(--primary-color-brand-01);
   font-weight: 700;
-  transition: border-bottom 1s;
-  border-bottom: 2px solid transparent;
-}
-
-.main-content-menu li:hover {
-  border-color: var(--primary-color-brand-01);
-}
-
-#burger-menu-check {
-  margin: 0;
-  height: 0;
-  width: 0;
-  visibility: hidden;
-  position: absolute;
-}
-#search {
   height: min-content;
   display: flex;
   background-color: transparent;
   align-items: center;
   justify-content: space-between;
-  transition: border-bottom 1s;
   padding: 35px 20px 18px;
   border: 0;
-  border-bottom: 2px solid transparent;
   gap: 10px;
   color: var(--primary-color-brand-01);
 }
-
-#search p {
-  font-size: 1rem;
-  text-transform: uppercase;
-  margin: 0;
-  font-weight: 700;
+/* This for draw the small line under the button */
+#search span {
+  position: relative;
+  transition: border-color 1s;
 }
-
+#search span::before {
+  content: " ";
+  width: 30px;
+  position: absolute;
+  border-bottom: 0.15rem solid transparent;
```

Comment:
use rem or px not both

Reply:
Can I use the "rem" unit for font size and "px" for everything else?